protocol/x/clob/types/codec.go

```diff
@@ -3,29 +3,12 @@ package types
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
-func RegisterCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterConcrete(&MsgProposedOperations{}, "clob/ProposedOperations", nil)
-	cdc.RegisterConcrete(&MsgPlaceOrder{}, "clob/PlaceOrder", nil)
-	cdc.RegisterConcrete(&MsgCancelOrder{}, "clob/CancelOrder", nil)
-	// this line is used by starport scaffolding # 2
-}
+func RegisterCodec(cdc *codec.LegacyAmino) {}
 
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
-	registry.RegisterImplementations((*sdk.Msg)(nil),
```

Comment:
Dumb question: I am having a hard time following why removing `RegisterImplementations` is okay. Can you point me to where in `RegisterMsgServiceDesc` we register the module msg types?

Reply:
https://github.com/dydxprotocol/cosmos-sdk/blob/6d0ef15b0a45078584f629ba14cb38ebe0fd0b81/types/msgservice/msg_service.go#L53

We use this method to iterate over the proto generated descriptor of the msg server and it automatically registers all of the messages for us (both the request and response messages)